server/base/src/test/java/org/apache/accumulo/server/conf/ZooBasedConfigurationTest.java

```diff
@@ -167,7 +167,7 @@ public void getPropertiesTest() {
     assertEquals("9998", zbc.get(GC_PORT));
 
     // read a property from the sysconfig
-    assertEquals("3", zbc.get(MANAGER_BULK_RETRIES));
+    assertEquals(5 * 60 * 1000, zbc.getTimeInMillis(MANAGER_BULK_TIMEOUT));
```

Comment:
For this kind of stuff, it's sometimes helpful to be explicit with something like:

```suggestion
    assertEquals(MINUTES.toMillis(5), zbc.getTimeInMillis(MANAGER_BULK_TIMEOUT));
```

Reply:
Applied your suggestion in 621910b